biomage/rds/tunnel.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+ENVIRONMENT=$1
+LOCAL_PORT=$2
+ENDPOINT_TYPE=$3
+
+function cleanup() {
+	ssh -O exit -S temp-ssh.sock *
+	rm temp*
+	echo "Finished cleaning up"
+}
+
+RDSHOST="$(aws rds describe-db-cluster-endpoints \
+	--region eu-west-1 \
```

Comment:
can you make sure that people can specify a different region (for multi-account deployments)?